src/python/pants/bin/daemon_pants_runner.py

```diff
@@ -279,10 +277,11 @@ def run(self):
                     session, specs, options, options_bootstrapper
                 )
                 with ExceptionSink.exiter_as_until_exception(lambda _: PantsRunFailCheckerExiter()):
-                    runner = LocalPantsRunner.create(
-                        self._args, self._env, specs, session, options_bootstrapper,
-                    )
-                    runner.set_start_time(self._maybe_get_client_start_time_from_env(self._env))
+                    runner = LocalPantsRunner.create(env, options_bootstrapper, specs, session)
+
+                    st = self._env.pop("PANTSD_RUNTRACKER_CLIENT_START_TIME", None)
```

Comment:
Maybe `hardcoded_start_time`? `st` took me a moment to figure out what it stood for.